crates/specs/src/imtable.rs

```diff
@@ -41,3 +76,931 @@ impl InitMemoryTable {
         self.0.get(&(ltype, offset))
     }
 }
+
+pub fn memory_event_of_step(event: &EventTableEntry) -> Vec<MemoryTableEntry> {
```

Comment:
If it’s necessary, could you move the function to `specs/src/mtable.rs`? I’m curious why it was moved to the `specs` crate.